doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1433,71 +1224,359 @@ This setting controls which upload methods are available to users of your instal
+
+``SiteFullName=Repository for Open Data``
+
+Setting can be postfixed with language code to obtain translated versions.
+
+``SiteFullName.pl=Repozytorium Otwartych Danych``
+
+MaximumEmbargoLength
+++++++++++++++++++++
+
+Sets maximum embargo length in months. By default it is disabled (set to 0).
+
+``MaximumEmbargoLength=0``
+
+ShowPrivacyPolicyFooterLink
++++++++++++++++++++++++++++
+
+Enables showing link to Privacy Policy page in the footer. By default is set to 'false', so link won't be shown.
+
+``ShowPrivacyPolicyFooterLink=false``
+
+ShowTermsOfUseFooterLink
+++++++++++++++++++++++++
+
+Enables showing link to Terms of Use page in the footer. By default is set to 'false', so link won't be shown.
+
+``ShowTermsOfUseFooterLink=false``
+
+JVM Options
+-----------
+
+JVM stands Java Virtual Machine and as a Java application, Glassfish can read JVM options when it is started. A number of JVM options are configured by the installer below is a complete list of the Dataverse-specific JVM options. You can inspect the configured options by running:
+
+``./asadmin list-jvm-options | egrep 'dataverse|doi'``
+
+When changing values these values with ``asadmin``, you'll need to delete the old value before adding a new one, like this:
+
+``./asadmin delete-jvm-options "-Ddataverse.auth.password-reset-timeout-in-minutes=10``
+
+``./asadmin create-jvm-options "-Ddataverse.auth.password-reset-timeout-in-minutes=30"``
+
+It's also possible to change these values by stopping Glassfish, editing ``glassfish4.1.2/glassfish/domains/domain1/config/domain.xml``, and restarting Glassfish.
+
+dataverse.auth.password-reset-timeout-in-minutes
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+Users have 60 minutes to change their passwords by default. You can adjust this value here.
+
+dataverse.files.directory
++++++++++++++++++++++++++
+
+This is how you configure the path to which files uploaded by users are stored.
+
+dataverse.path.imagemagick.convert
+++++++++++++++++++++++++++++++++++
+
+For overriding the default path to the ``convert`` binary from ImageMagick (``/usr/bin/convert``).
+
+dataverse.dataAccess.thumbnail.image.limit
+++++++++++++++++++++++++++++++++++++++++++
+
+For limiting the size (in bytes) of thumbnail images generated from files.
+
+dataverse.dataAccess.thumbnail.pdf.limit
+++++++++++++++++++++++++++++++++++++++++
+
+For limiting the size (in bytes) of thumbnail images generated from files.
+
+dataverse.lang.directory
+++++++++++++++++++++++++
+
+This JVM option is used to configure the path where all the language specific property files are to be stored.  If this option is set then the english property file must be present in the path along with any other language property file.
+
+``./asadmin create-jvm-options '-Ddataverse.lang.directory=PATH_LOCATION_HERE'``
+
+If this value is not set, by default, a Dataverse installation will read the English language property files from the Java Application.
+
+Database Settings
```

Comment:
Yeah but since they are not in the file since we didn't move them, wouldn't it be weird for the user to find out that they are missing? Shouldn't we move them first ourselves?

Reply:
Well. We could move them.